front/src/routes/integration/all/nuki/NukiPage.js

```diff
@@ -0,0 +1,71 @@
+import { Text } from 'preact-i18n';
+import { Link } from 'preact-router/match';
+
+const NukiPage = ({ children }) => (
+  <div class="page">
+    <div class="page-main">
+      <div class="my-3 my-md-5">
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-3">
+              <h3 class="page-title mb-5">
+                <Text id="integration.nuki.title" />
+              </h3>
+              <div>
+                <div class="list-group list-group-transparent mb-0">
```

Comment:
The links order is weird, when clicking on the integration, I arrive on the "device tab", but it's not the first 🤔

![Screenshot 2025-05-08 at 09 11 59](https://github.com/user-attachments/assets/7535acba-7cb1-4b84-9ef9-02ddb71342cd)

1. Capitalize the "nuki" title
2. I would put the following order:
   1. Appareils
   2. Découverte MQTT
   3. Découverte Web
   4. Configuration
   5. Documentation (should be the link to the documentation, you'll see this pattern in many other integration, we have a special component for that)